packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -947,8 +947,10 @@ class _NestedScrollPosition extends ScrollPosition implements ScrollActivityDele
   // Returns the amount of delta that was not used.
   double applyClampedDragUpdate(double delta) {
     assert(delta != 0.0);
-    final double min = delta < 0.0 ? -double.INFINITY : minScrollExtent;
-    final double max = delta > 0.0 ? double.INFINITY : maxScrollExtent;
+    // Positive delta means going down (exposing stuff above), negative delta
+    // going up (exposing stuff below).
+    final double min = delta < 0.0 ? -double.INFINITY : math.min(minScrollExtent, pixels);
```

Comment:
First time looking at this code, tried to snoop around and understand what's going on a bit, but I'm still lost.

Could you add a comment explaining the fix? the questions I'm currently trying to understand are: when will pixels be out of the min/max scroll extent? IIUC with the new change here in the case pixels is already out of the min/max extent we just ignore a drag that makes it go even farther away from the min/max scroll extent? why is that what we want to do?

(I'm totally aware that it is very likely that I completely misunderstood what's going here :) )